simulation/CMakeLists.txt

```diff
@@ -77,6 +78,7 @@ find_package(JsonCpp REQUIRED)
 include_directories(
   include
   ${config_reader_INCLUDE_DIRS}
+  ${pmc_INCLUDE_DIRS}
```

Comment:
need to figure out why this was needed, I set the ament_export_include_directories and added it to the ament_target_dependencies, but it still complained that it couldn't find #include <pmc/pmc_sim.h>

Reply:
I'm reading in the ament documentation that to export the header files we should use this:

```
  target_include_directories(my_target
    PUBLIC
      $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
      $<INSTALL_INTERFACE:include>)
```

https://docs.ros.org/en/humble/How-To-Guides/Ament-CMake-Documentation.html

Might want to try that in the pmc CMakeLists